docs/source/programming_resources/shared/program_and_manage_network/Connecting-a-Laptop-to-the-Program-&-Manage-Network.rst

```diff
@@ -0,0 +1,131 @@
+installed.
+
+Note that this example assumes the user has a Windows 10 laptop. If you
+are not using a Windows 10 laptop, the procedure to connect to the
+Programming & Manage Wi-Fi network will differ. Refer to your device's
+documentation for details on how to connect to a Wi-Fi network.
+
+Connecting Your Laptop to the Program & Manage Network Instructions
+-------------------------------------------------------------------
+
+1. On the Driver Station, touch the three dots in the upper right     
+hand corner of the screen to launch the pop-up menu. Select **Program 
+& Manage** from the pop-up menu to display the **Program & Manage**   
+access information.                                                   
+
+.. image:: images/SelectProgramAndManageDriverStation.jpg
+   :align: center
+
+|
+
+2. The Program & Manage screen displays important information that    
+you can use to connect your laptop to the FTC Blocks or OnBot Java    
+Programming Mode server.                                              
+
+.. image:: images/ProgramAndManageScreen.jpg
+   :align: center
+
+|
+
+3. Verify the network name and passphrase for the Program & Manage    
+wireless network. Towards the top of the screen, the name of the      
+Program & Manage wireless network is displayed. If you are using an   
+Android smartphone as your Robot Controller, then the wireless        
+network name will begin with the phrase "DIRECT-". 
+
+   In this example, the name of the Wi-Fi network is "DIRECT-XK-9999-C-RC" and the secure passphrase is "ZU7if0hB"                                              
+
+.. image:: images/ProgramAndManagePassphrase.jpg
+   :align: center
+
+|
+
+If you are using a Control Hub, then the wireless network name will be whatever you specified when you configured your Control Hub.  If you haven't changed the Control Hub's name yet, then by default the  wireless network's name will begin with "FTC-".  If you haven't changed its password yet, then by default the wireless network's passphrase will be "password".
+
+In the screenshot below, the Control Hub's wireless network name is "FTC-1Ybr" and the secure passphrase is "password".
+
+.. image:: images/ProgramAndManagePassphraseControlHub.jpg
+   :align: center
+
+|
+
+4. On your Windows 10 computer, look in the lower right hand corner   
+of your desktop for a Wi-Fi symbol. Click on the Wi-Fi symbol to      
+display a list of available Wi-Fi Networks in your vicinity.          
+
+.. image:: images/ConnectingLaptopStep4.jpg
+   :align: center
+
+|
+
+5. Look for the wireless network that matches the name displayed on   
+the Program & Manage screen.                                          
+
+.. image:: images/screengrabwirelessnetworks.jpg
+   :align: center
+
+|
+
+   In this example, the name of the wireless network for the Android Robot Controller is "DIRECT-XK-9999-C-RC" and the network is visible in the list displayed on the Windows 10 computer.
```

Comment:
```
`DIRECT-XK-9999-C-RC`
```

Reply:
I think this goes beyond the scope of the conversion.